site/src/_layouts/component.hbs

```diff
@@ -37,6 +37,14 @@ layout: default
 
     <nav class="col-sm-3 hidden-xs">
       <ul id="sidebar" class="nav nav-stacked fixed">
+        <li>
```

Comment:
I know it's a bit of a corruption but any reason not to just do these as components and add them to the ```_config.yml```?